package.json

```diff
@@ -57,15 +58,19 @@
     "format": "npm run prettier:fix && npm run lint:fix"
   },
   "devDependencies": {
+    "@tableland/local": "^1.2.3",
     "@types/crypto-js": "^4.1.1",
     "@types/mocha": "^10.0.1",
     "@typescript-eslint/eslint-plugin": "^5.59.6",
     "@typescript-eslint/parser": "^5.59.6",
+    "c8": "^7.13.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-config-standard": "^17.0.0",
     "mocha": "^10.0.0",
+    "mockipfs": "npm:@joewagner/mockipfs@^0.3.2",
```

Comment:
I have an open PR to get these changes into the official package, but until then I published a version that works.